snapcraft/plugins/gradle.py

```diff
@@ -104,7 +118,7 @@ def build(self):
             basedir = 'war'
             jarfiles = warfiles
         else:
-            raise RuntimeError("Could not find any built jar files for part")
+            raise RuntimeError("Could not find any built jar/war files in the part directory:" + src)
```

Comment:
Please use `.format()` here instead of appending (preferably with `{!r}`). Also, please update this to use single quotes.

Reply:
I will push an update shortly.